app/questions.py

```
@@ -0,0 +1,340 @@
+    id: int
+    tamil_numeral: str
+    tamil: str
+    transliteration: str
+    english: str
+    is_gate: bool
+    sangam_ref: Optional[str] = None
+
+
+QUESTIONS: list[Question] = [
+    # ═══════════════════════════════════════════════════════════════
+    # வாயில் — THE GATE (Questions 0–1)
```

Ruff (0.15.4)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)